Makefile

```diff
@@ -285,6 +288,12 @@ docker: build-binaries-linux docker-images-only
 build-binaries-linux:
 	GOOS=linux GOARCH=amd64 $(MAKE) build-platform-binaries
 
+.PHONY: build-binaries-linux-archs
+build-binaries-linux-archs:
```

Comment:
I see that we have added this new directive. Why didn't we update the existing `build-binaries-linux` directive instead since that is being called as dependencies everywhere ?

Reply:
Now just for local build.
This pr should now change too lot for CI.